scripts/gfg.js

```diff
@@ -13,24 +13,24 @@ const UPDATE_MSG = 'Updated solution - LeetHub';
 let START_MONITOR = true;
 const toKebabCase = (string) => {
   return string
-    .replace(/[^a-zA-Z0-9\. ]/g, '') //remove special chars
+    .replace(/[^a-zA-Z0-9\. ]/g, '') // remove special chars
     .replace(/([a-z])([A-Z])/g, '$1-$2') // get all lowercase letters that are near to uppercase ones
     .replace(/[\s_]+/g, '-') // replace all spaces and low dash
     .toLowerCase(); // convert to lower case
 };
 
 function findGfgLanguage() {
-  const ele = document.getElementsByClassName('filter-option')[0]
+  const ele = document.getElementsByClassName('divider text')[0]
     .innerText;
-  let lang = ele.split('(')[0].trim();
+  const lang = ele.split('(')[0].trim();
   if (lang.length > 0 && languages[lang]) {
     return languages[lang];
   }
   return null;
 }
 
 function findTitle() {
-  const ele = document.getElementsByClassName('problem-tab__name')[0]
+  const ele = document.querySelector('[class^="problems_header_content__title"] > h3')
```

Comment:
could you define the set of changes in the PR so its easier for me to take a look at? i see this is one of the class names to be changed.

Reply:
Update the required selectors as per the new gfg site.